scripts/build-toolchains.sh

```diff
@@ -134,7 +134,7 @@ SRCDIR="$(pwd)/toolchains" module_all qemu --prefix="${RISCV}" --target-list=ris
 git submodule update --init $CHIPYARD_DIR/tools/dromajo/dromajo-src
 make -C $CHIPYARD_DIR/tools/dromajo/dromajo-src/src
 
-cd "$RDIR"
+cd "${CHIPYARD_DIR}"
```

Comment:
Isn't this always going to return you to the top-level? The point of `cd $RDIR` was to take you back to where you were when you sourced the script

Reply:
you shouldn't source this script though

Reply:
Yeah, good point. What's this change doing, then?

Reply:
there is actually no reason this line needs to exist in the first place. i should actually just remove it

Reply:
i lied. originally there was a `ln env-$TOOLCHAIN.sh env.sh` at the end of the script, which requires you to cd to $CHIPYARD_DIR to work. but that `ln` command no longer exists, so we don't need to `cd` anymore